src/controller/review.py

```diff
@@ -15,6 +15,10 @@ def doCSV():
 
 @review.route('/review')
 def getApplicant():
+
+    if ApplicationsModel.ApplicationsModel.total_count == ApplicationsModel.ApplicationsModel.done_count:
+        return Response(status=300)
```

Comment:
I don't think 300 is the appropriate status here. Maybe 404?

Reply:
I feel like we should send some sort of message, 404 is like the route isn't there